### DISCUSSION TITLE
No mod ratios in csv file

### DISCUSSION THREAD

**Discussion by esh97** [GoekeLab/m6anet#62] [Q&A] [answered]
Hello,

I am running the program on my data set, and it is able to generate the csv file but in the modification columns it only outputs a 0. Am I doing some wrong in my data prep upstream of the analysis?

I start by indexing my gzipped fast5 files with the fastq files using nanopolish index, then I align using eventalign followed by the data prep and mod inference modules of m6anet.

Thanks!

**Comment by chrishendra93** [comment]
hi @esh97 

Can you show me the first 5 rows of your nanopolish eventalign.txt and your result files?

Thanks!

**Reply by esh97** [reply]
Thanks for your quick response!

Here are the eventalign.txt and modification results:

![image](https://user-images.githubusercontent.com/105374892/205931083-4169edb6-75c6-4334-9128-3c3c90b31884.png)

![image](https://user-images.githubusercontent.com/105374892/205931441-ee1b3c62-f3ad-4b3b-8b83-2ba493736def.png)

**Comment by chrishendra93** [comment, marked as answer]
Hi @esh97 , your eventalign file does not have the start_index and end_index column that tells m6anet how to segment your reads. When running nanopolish make sure you include the -signal-index flag such as:

nanopolish eventalign –reads reads.fastq –bam reads.sorted.bam –genome transcript.fa –scale-events –signal-index –summary 
        /path/to/summary.txt –threads 50 > /path/to/eventalign.txt

**Reply by esh97** [reply]
Thank you! That worked
